src/lib/sys.c

```diff
@@ -286,6 +286,22 @@ bool sys_file_exists(const char *filename)
 	return false;
 }
 
+bool sys_absolute_path_exists(const char *filename)
```

Comment:
I think that the most appropriate name for this function would be `sys_path_is_absolute()` because it's not checking only if the path exists.

Reply:
nice, I was unsure about the name of the function and I was going to ak you for your opinion. `sys_path_is_absolute()` sounds good to me.